main_window/data_entry_id.py

```diff
@@ -45,18 +45,32 @@ class DataEntryIds(Enum):
 
 
 class DataEntryValues(Enum):
-    NOMINAL = auto()
-    NONCRITICAL_FAILURE = auto()
-    CRITICAL_FAILURE = auto()
-
-    STANDBY = auto()
-    ARMED = auto()
-    ASCENT = auto()
-    MACH_LOCK = auto()
-    PRESSURE_DELAY = auto()
-    INITIAL_DESCENT = auto()
-    FINAL_DESCENT = auto()
-    LANDED = auto()
-    WINTER_CONTINGENCY = auto()
-
-    # TODO Add Events, other info stored
+    STATE_NOMINAL = auto()
+    STATE_NONCRITICAL_FAILURE = auto()
+    STATE_CRITICAL_FAILURE = auto()
```

Comment:
Technically these are for "status" not "state"
```suggestion
    STATUS_NOMINAL = auto()
    STATUS_NONCRITICAL_FAILURE = auto()
    STATUS_CRITICAL_FAILURE = auto()
```

Reply:
CI is failing. This still needs to be renamed somewhere else in the code.